From e1a760f6aa8d87b54ace71a4a371f74a4d26a70e Mon Sep 17 00:00:00 2001
From: Ke Qinghua <qinghua.ke@freescale.com>
Date: Fri, 20 Dec 2013 15:13:20 +0800
Subject: [PATCH 273/602] ENGR00292372 Add consumer IR support in android 4.4

Add marco in board config for consumer IR enable/disable

Signed-off-by: Ke Qinghua <qinghua.ke@freescale.com>
---
 sabresd_6dq/BoardConfig.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/sabresd_6dq/BoardConfig.mk b/sabresd_6dq/BoardConfig.mk
index 2abd908..7c7a67a 100755
--- a/sabresd_6dq/BoardConfig.mk
+++ b/sabresd_6dq/BoardConfig.mk
@@ -103,6 +103,8 @@ IMX_CAMERA_HAL_V2 := true
 # define frame buffer count
 NUM_FRAMEBUFFER_SURFACE_BUFFERS := 3
 
+#define consumer IR HAL support
+IMX6_CONSUMER_IR_HAL := true
 
 TARGET_BOOTLOADER_CONFIG := 6q:mx6q_sabresd_android_config 6dl:mx6dl_sabresd_android_config
 
-- 
1.8.0

